.github/workflows/CI.yaml

```diff
@@ -25,7 +25,15 @@ jobs:
       fail-fast: false
       matrix:
         os: [macOS-latest, ubuntu-latest, windows-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        include:
+          - name: numpy_min
+            os: ubuntu-latest
+            python-version: 3.6
+            run_type: FULL
```

Comment:
What is `FULL`? I don't think you use this line

Reply:
Thanks, I missed this. Same below.